koku/masu/processor/expired_data_remover.py

```diff
@@ -154,4 +157,10 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
                 removed_data = self._cleaner.purge_expired_line_item(expired_date=expiration_date, simulate=simulate)
             else:
                 removed_data = self._cleaner.purge_expired_report_data(expired_date=expiration_date, simulate=simulate)
+                if not simulate:
+                    CostUsageReportManifest.objects.filter(
```

Comment:
We talked about using the ReportManifestDBAccessor for this. Also didn't we say we wanted to support simulate and have it tell us how many records would have been deleted if it had been real?

Reply:
@adberglund added

Reply:
@adberglund, given @myersCody recent comment below, do you agree that simulate should only show a generic message not not show how many records were deleted?

Reply:
Yes, that sounds just fine.

Reply:
@adberglund added